boards/FSM/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -143,7 +147,69 @@ void App_SetPeriodicSignals_AcceleratorPedal(const struct FsmWorld *world)
 
 void App_SetPeriodicSignals_MotorShutdownFaults(const struct FsmWorld *world)
 {
-    struct FsmCanTxInterface *can_tx = App_FsmWorld_GetCanTx(world);
+    struct FsmCanTxInterface *can_tx      = App_FsmWorld_GetCanTx(world);
+    struct FsmCanRxInterface *can_rx      = App_FsmWorld_GetCanRx(world);
+    static uint8_t            error_count = 0;
+
+    // Perform torque plausibility check if DCM is in the drive state
+    if (App_CanRx_DCM_STATE_MACHINE_GetSignal_STATE(can_rx) ==
+        CANMSGS_DCM_STATE_MACHINE_STATE_DRIVE_CHOICE)
```

Comment:
We can check whether we are in the air closed state instead of receiving this message from the DCM

Reply:
I think we should keep this